code/modules/projectiles/gun.dm

```diff
@@ -49,6 +49,7 @@
 	origin_tech = list(TECH_COMBAT = 1)
 	attack_verb = list("struck", "hit", "bashed")
 	zoomdevicename = "scope"
+	action_button_name = ""

```

Comment:
Is this necessary? The default value should do, IMO, ergo, remove the needless override.